homepage.css

```diff
@@ -0,0 +1,98 @@
+.homepage-container{
+    padding: 0;
+    margin: 0;
+    display: flex;
+    align-items: center;
+    overflow: hidden;
+    /* background: linear-gradient(to right, #75a7c6, #d88779); */
+}
+.navbar{
+    padding: 0.2rem;
+ }
+ .search-container{
+    flex-basis: 26%;
+    border-radius: 23px;
+    background: var(--white)
+ } 
+.search-bar{
+    outline: none;
+    width: 100%;
+    border: none;
+    font-size: 1rem;
+    padding: 0.6rem 0.8rem;
+    margin: 0.5rem;
+}
+.fa-search{
+    color: var(--primary);
+    margin-right: 10px;
+}
+
+/* .navbar-styling
+
+.slider-frame{
+    overflow: hidden;
+    height: 1000px;
+    width: 1200px;
+    margin-left: 360px;
+    margin-top: 300px;
+}
+@-webkit-keyframes slide_animation{
+    0%{left:0px;}
+    10%{left:0px;}
+    20%{left:1200px;}
+    30%{left:1200px;}
+    40%{left:2400px;}
+    50%{left:2400px;}
+    60%{left:1200px;}
+    70%{left:1200px;}
+    80%{left:0px;}
+    90%{left:0px;}
+    100%{left:0px;}
+}
+
+.slide-images{
+    width:3600px;
+    height: 800px;
+    margin: 0 0 0 -2400px;
+    position: relative;
+    -webkit-animation-name: slide_animation ;
+    -webkit-animation-duration: 15s;
+    -webkit-animation-iteration-count:infinite;
+    -webkit-animation-direction: alternate;
+    -webkit-animation-play-state:running ;
+}
+
+.img-container{
+    height: 1000px;
```

Comment:
You can remove commented part

Reply:
> You can remove commented part

thank you for bringing this into my notice, will do this!